app/actions/trainer.js

```diff
@@ -527,8 +533,7 @@ function batchProcessSelectedPokemon(method, batchMethod, selectedPokemon) {
   }
 }
 
-const transferSelectedPokemon = batchProcessSelectedPokemon.bind(null, 'Transfer', 'releasePokemon')
-
+const transferSelectedPokemon = transferPokemon.bind('releasePokemon')
```

Comment:
You can remove the `.bind()` call here, you don't use the param in the method above.